flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -277,7 +277,8 @@ protected static Notification createNotification(
             .setContentIntent(pendingIntent)
             .setPriority(notificationDetails.priority)
             .setOngoing(BooleanUtils.getValue(notificationDetails.ongoing))
-            .setOnlyAlertOnce(BooleanUtils.getValue(notificationDetails.onlyAlertOnce));
+            .setOnlyAlertOnce(BooleanUtils.getValue(notificationDetails.onlyAlertOnce))
+            .setSilent(BooleanUtils.getValue(notificationDetails.silent));
```

Comment:
Nit: you might want to write this below `ONGOING` to keep the order with the rest

Reply:
Fixed :)